packages/kit/src/views/Onboarding/pages/ConnectHardwareWallet/ActivateDeviceResource.ts

```
@@ -1,97 +1,104 @@
+    case EDeviceType.Mini:
       return require('@onekeyhq/kit/assets/onboarding/mini-enter-recovery-phrase.png');
-    case 'touch':
+    case EDeviceType.Touch:
       return require('@onekeyhq/kit/assets/onboarding/touch-enter-recovery-phrase.png');
-    case 'pro':
+    case EDeviceType.Pro:
       return require('@onekeyhq/kit/assets/onboarding/pro-enter-recovery-phrase.png');
     default:
       // eslint-disable-next-line no-case-declarations, @typescript-eslint/no-unused-vars
-      const _exhaustiveCheck: never = type;
+      const _exhaustiveCheck = type;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)